repository-data/application/src/main/resources/hcm-config/namespaces/publicationsystem/publication.yaml

```diff
@@ -244,6 +247,8 @@ definitions:
             hipposysedit:path: publicationsystem:CoverageEnd
             hipposysedit:primary: false
             hipposysedit:type: CalendarDate
+            hipposysedit:validators:
+            - publicationsystem-coverage-dates
```

Comment:
Does `dataset.yaml` need the same validation applied?

Reply:
Will add